app/components/views/TicketsPage/PurchaseTab/StakePools/AddForm.js

```diff
@@ -24,72 +24,79 @@ const StakePoolsAddForm = ({
-
-            1) Create an account or login to your existing account at {stakePoolLink}.
-            2) Once logged in, select the 'Settings' tab.
-            3) Copy and paste your Api Key into the field below (typically starts with 'eyJhb...').
-            4) Click Add and enter your private passphrase.
-
-            {noticeSpan} If you receive an error about the script not being redeemable when attempting to add your stakepool, you can try the following:
-            - Each stakepool account you create can only be associated with 1 wallet.  If you have previously created this stakepool account with a different wallet (different seed), then you must create a new account.
-            - If you had previously used a 'voting account', for your ticket purchases, please go to the Accounts page and create a new account.  This may now allow you to successfully import your script for your stakepool.
-            `}
-          values={{
-            stakePoolLink: <a className="stakepool-link" onClick={function(x){shell.openExternal(x);}.bind(null, selectedUnconfigured.label)}>{selectedUnconfigured.label}</a>,
-            noticeSpan: <span className="stakepool-highligh-text-orange">
-              <T id="stake.addPool.notice" m="Notice!" />
-            </span>
-          }}/>
-        </span>
-      </div>
-      <div className="stakepool-content-nest-to-address">
-        <div className="stakepool-content-nest-api-key">
-          <div className="stakepool-input-form">
+        <div className="stakepool-field">
+          <div className="stakepool-field-label">
+            <T id="stakepool.apikey" m="API Key" />:
+          </div>
+          <div className="stakepool-field-value">
             <TextInput
               type="text"
-              className="stakepool-content-nest-address-amount-sum"
+              className="stakepool-add-apikey"
               placeholder={intl.formatMessage(messages.apiKeyPlaceholder)}
               value={apiKey}
               onChange={e => onChangeApiKey(e.target.value)}
             />
           </div>
+          <div className="stakepool-field-value-error">
+            {apiKey ? null : <T id="stake.addPool.errors.noApiKey" m="*Please enter your API key" /> }
+          </div>
         </div>
-        <div className="stakepool-api-key-error">
-          {apiKey ? null : <T id="stake.addPool.errors.noApiKey" m="*Please enter your API key" /> }
+      </div>
+      <div className="stakepool-add-area-right">
+        <div className="stakepool-add-instructions">
+          <T id="stake.addPool.info" m={
+            "Create an account or login to your existing account at {stakePoolLink} Once logged in, select the ‘Settings’ tab, copy and paste your API KEY into the field."
+          }
+          values={{
+            stakePoolLink: <a className="stakepool-link" onClick={function(x){shell.openExternal(x);}.bind(null, selectedUnconfigured.label)}>{selectedUnconfigured.label}</a>
```

Comment:
Instead of manually using the onClick={shel...}  use an `ExternalLink` component (with href to the url).